.gitignore

```diff
@@ -4,3 +4,4 @@ dist
 .idea/
 *.egg-info
 /.tox
+venv/
```

Comment:
This should be part of your .git/info/exclude, not in tree.

Reply:
Oh sorry, I'll remove it